extensions/industry/healthcare/fhir/datatypes/repeat.schema.json

```diff
@@ -0,0 +1,168 @@
+{
+   "meta:license":[
+      "Copyright 2024 Adobe Systems Incorporated. All rights reserved.",
+      "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+      "you may not use this file except in compliance with the License. You may obtain a copy",
+      "of the License at https://creativecommons.org/licenses/by/4.0/"
+   ],
+   "$schema":"http://json-schema.org/draft-06/schema#",
+   "$id":"https://ns.fhir.org/datatypes/repeat",
+   "title":"Repeat",
+   "type":"object",
+   "description":"A set of rules that describe when the event is scheduled.(This datatype is created as per HL7 FHIR Release 5 specifications)",
+   "definitions":{
+      "repeat":{
+         "properties":{
+            "xdm:boundsdDuration":{
+               "title":"Bound Duration",
+               "description":"Duration Limit",
+               "type":"string"
+            },
+            "xdm:boundsRange":{
+               "title":"Bound Range",
+               "description":"Range Limit",
+               "$ref":"https://ns.fhir.org/datatypes/range"
+            },
+            "xdm:boundsPeriod":{
+               "title":"Bound Period",
+               "description":"Start and End Times",
+               "$ref":"https://ns.fhir.org/datatypes/period"
+            },
+            "xdm:count":{
+               "title":"count",
+               "description":"Number of times to repeat",
+               "type":"integer"
```

Comment:
Let's add `"minimum": 0` since it is a positive integer

Reply:
sure!